crates/pop-cli/src/common/helpers.rs

```diff
@@ -0,0 +1,302 @@
+// SPDX-License-Identifier: GPL-3.0
+
+use anyhow;
+use std::{
+	fs::{read_to_string, write},
+	path::{Path, PathBuf},
+};
+use toml_edit::{value, DocumentMut, Item, Value};
+
+/// This function is used to determine if a Path is contained inside a workspace, and returns a PathBuf to the workspace Cargo.toml if found
```

Comment:
Just for consistency, we use this format for comments, example: https://github.com/r0gue-io/pop-cli/blob/main/crates/pop-common/src/manifest.rs#L7